packages/common/src/core/traits/IWithViewSettings.ts

```diff
@@ -58,4 +59,7 @@ export interface IWithViewSettings {
    * array of actions for action links
    */
   heroActions?: HubActionLink[];
+
+  /** array of embedded content */
+  embeds?: HubEmbed[];
```

Comment:
Is the assumption that the order of the embeds === order injected into the view? Or should we have a `key` prop on `IHubEmbedBase` so that we can fetch specific embed's out of the array?